Cue/app/reducers/discover.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import type {Action} from '../actions/types';
+import type {DeckMetadata} from '../api/types';
+
+export type State = {
+  searchString: string;
+  searchResults: Array<DeckMetadata>;

```

Comment:
use type `?Array<DeckMetadata>` since you are setting it to `null` in `initalState`, or set `initialState=[]`